configure

```diff
@@ -298,8 +300,14 @@ if ! sh ./config.status; then
     exit 1
 fi
 
+if type gmake > /dev/null 2>&1 ; then
+    gmake=gmake
+else
+    gmake=make
+fi
+
 # Create dirs
-make create-dirs
+${gmake} create-dirs
 
```

Comment:
Let's do the same for make: allowing to use MAKE=gmake

Reply:
Yes that is indeed a more general solution.